eZ/Publish/Core/REST/Client/Values/User/User.php

```diff
@@ -35,6 +35,14 @@ public function getVersionInfo()
     }
 
     /**
+     * {@inheritdoc}
```

Comment:
> I wouldn't use {@inheritDoc} here. (...) At the moment, PhpDocumentor doesn't know about the non-inline usage of that tag. 

Hmm, not sure what to think about. I don't like redundancy and also PhpStorm displays proper doc here. However if PhpDocumentor has problems with it and it does not conform to standard, then, maybe, I agree with you... There's a bigger issue. We (including myself) are using this in a lot of places... :)